pkg/plugins/resources/postgres/store.go

```diff
@@ -127,6 +127,20 @@ func (r *postgresResourceStore) Delete(_ context.Context, resource model.Resourc
 	return nil
 }
 
+func (r *postgresResourceStore) DeleteMany(ctx context.Context, fs ...store.DeleteManyOptionsFunc) error {
+	opts := store.NewDeleteManyOptions(fs...)
+	statement := `DELETE FROM resources`
+	var statementArgs []interface{}
+	if opts.Mesh != "" {
```

Comment:
Support for `opts.Mesh == ""` is too dangerous (I don't think somebody will ever need this feature)